### PR TITLE
chore: remove old wallet code, signing code and keycard code

### LINKED ISSUE #18559: Wallet: Delete Legacy Wallet code
After 1.27 release is made the old wallet will be completely removed from the UI.
The legacy code then becomes redundant.

Go through the legacy folders and remove the old wallet.
this includes configs etc.

- following on from this:
- [x] it includes all legacy wallet ui code
- [x] all legacy wallet subs
- [x] all legacy wallet events
- [ ] legacy signing
- [ ] legacy keycard

### PR DESCRIPTION
**Pull request by J-Son89**
fixes: https://github.com/status-im/status-mobile/issues/18559

This pr deletes numerous other parts of the legacy code. I could not find any of these screens as a result of the legacy wallet already being deleted.

This also removes the keycard code as we are currently not supporting the legacy implementation anyway and the new implementation will have to be revisited.

It removes all signing and wallet connect pages for the same reason as keycard.

To test:
A quick happy path pass through the app and check some of the legacy settings particularly related to wallet/ network and keycard signing etc. 
It is also good if a quick check of the browser can be done.

### CODE REVIEW

File: src/status_im/config.cljs [hunk 1]
```diff
@@ -99,62 +99,6 @@
   [chain]
   (get-in default-multiaccount [:wallet-legacy/visible-tokens chain]))
 
-(def mainnet-networks
```
Comment (J-Son89) [inline src/status_im/config.cljs:107]: I think it's safe to delete all these @smohamedjavid

File: src/status_im/contexts/profile/login/events.cljs [hunk 2]
```diff
@@ -45,12 +45,10 @@
...
-                      (assoc :chats/loading?           true
-                             :networks/current-network current-network
-                             :networks/networks        (merge networks config/default-networks-by-id)
-                             :profile/profile          (merge profile-overview
-                                                              settings
-                                                              {:log-level log-level}))
+                      (assoc :chats/loading?  true
+                             :profile/profile (merge profile-overview
+                                                     settings
+                                                     {:log-level log-level}))
```
Comment (OmarBasem) [inline src/status_im/contexts/profile/login/events.cljs:52]: These network keys not required anymore?

**Review by OmarBasem — APPROVED** [review]

File: src/status_im/contexts/profile/login/events.cljs (hunk 2 above)
Comment (J-Son89) [reply src/status_im/contexts/profile/login/events.cljs:52]: We shouldn't need them afaik. We put all the related network pieces into :wallet/networks as far as I know. There might be some code touching off these that I will double check, but realistically we should not be using any legacy subs so it's better we move away from this 👍

File: src/status_im/config.cljs (hunk 1 above)
Comment (smohamedjavid) [reply src/status_im/config.cljs:107]: Yes! We don't use this anymore.

File: src/status_im/contexts/profile/login/events.cljs (hunk 2 above)
Comment (smohamedjavid) [reply src/status_im/contexts/profile/login/events.cljs:52]: Yeah, `:networks/networks` contains the L1 chains data (used in the old wallet) which is outdated now.

File: src/status_im/contexts/profile/config.cljs [hunk 3]
```diff
@@ -40,9 +40,6 @@
             :verifyENSURL             config/verify-ens-url
             :verifyENSContractAddress config/verify-ens-contract-address
             :verifyTransactionChainID config/verify-transaction-chain-id
-            :upstreamConfig           config/default-network-rpc-url
-            :networkId                config/default-network-id
```
Comment (smohamedjavid) [inline src/status_im/contexts/profile/config.cljs:44]: I see a few usages of `networkId` in `status-go`. But, removing this doesn't affect it as it sets the default value (Chain ID `1`).

**Review by smohamedjavid — APPROVED** [review]
💣

**Review by qfrank — APPROVED** [review]
Time flies; I hope the new code will remain for a longer period. 🙂

**Comment by VolodLytvynenko** [comment]
Hi @J-Son89 thank you for PR. No issues from my side. Ready to be merged

**Pull request merged**